akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -127,6 +127,7 @@ private[akka] object NettySSLSupport {
           val sslEngine = context.createSSLEngine
           sslEngine.setUseClientMode(true)
           sslEngine.setEnabledCipherSuites(settings.SSLEnabledAlgorithms.toArray)
+          sslEngine.setEnabledProtocols(settings.SSLProtocol.toArray)
```

Comment:
without this line it would ignore the ssl-protocol setting, and pick up whatever was defined in env variables (defaulting to accepting all versions)